### PR TITLE
Reviewing Old Survey Project

### PR DESCRIPTION
**Pull request by jpol01**
## Reviewing Old Survey Project

Objective >
Add comments on our own project, so we can practice the code reviewer guidelines.

### CODE REVIEW

File: microverse3-trial-survey-main/@css/surcey_css.css [hunk 1]
```diff
@@ -108,7 +108,7 @@ button {
   text-align: center;
 }
 
-/* FORM CSS Selectors */
+/* FORM css Selectors */
```
Comment (jpol01) [inline microverse3-trial-survey-main/@css/surcey_css.css:112]: Is really good to leave comments and be specific, you did it very well :+1: 

 However, in some cultures, a sentence in full capital letters could be taken as you are screaming. So please change it to lowercase. :1st_place_medal:

**Review by jpol01 — COMMENTED** [review]
Is a great job, please check the comments, do the changes and you are ready to go. :grin:

File: microverse3-trial-survey-main/@css/surcey_css.css (hunk 1 above)
Comment (julie-ify) [reply microverse3-trial-survey-main/@css/surcey_css.css:112]: - [ ] Nice job on the wonderful review. Also, well done on the utilization of Github review features and inline comments features. However, you are required to add at least two reviews but I am currently seeing only one review. Consider going through your work and looking for where you can improve your code according to [HTML & CSS best practices ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) and how well you used [semantic HTML tags](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)😉.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @jpol01, 

Good job so far!

Highlight!
- Good feature branch name ✔️ 
- No linter errors ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️⛔️
_Check the comments under your review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: microverse3-trial-survey-main/@css/surcey_css.css [hunk 2]
```diff
@@ -108,7 +108,7 @@ button {
   text-align: center;
 }
 
-/* Form css Selectors */
+/* Css Selectors Form */
```
Comment (jpol01) [inline microverse3-trial-survey-main/@css/surcey_css.css]: The change of the comment was accurate, thanks for applying the change. Well done :+1:

File: microverse3-trial-survey-main/@css/surcey_css.css [hunk 3]
```diff
@@ -22,7 +22,7 @@ body {
   margin: 0;
 }
 
-/* mobile friendly alternative to using background-attachment: fixed */
+/* Mobile friendly alternative to using background-attachment: fixed */
 body::before {
```
Comment (jpol01) [inline microverse3-trial-survey-main/@css/surcey_css.css:26]: Applying good practices by adding the first capital letter. Well done.

**Review by elyor-doniyorov — APPROVED** [review]
Hi @jpol01, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**